src/pages/Embark/createQuote.ts

```diff
@@ -53,24 +52,12 @@ const MUTATION = gql`
   }
 `
 
-export const createQuote = (storage: any) => async (
+export const createQuote = (storage: any, locale: Locale) => async (
   variables: CreateQuoteVariables,
 ): Promise<CreateQuoteData | Error> => {
-  if (!apolloClient) {
-    return Error('Missing apollo client')
-  }
+  await setupSession(apolloClient!, storage, locale)
 
-  if (!storage.session.getSession().token) {
-    const sessionResult = await apolloClient.client.mutate({
-      mutation: CREATE_SESSION_TOKEN_MUTATION,
-    })
-    await afterTick(() => {
-      apolloClient!.subscriptionClient.close(true, true)
-      storage.setToken(sessionResult.data.createSessionV2.token)
-    })
-  }
```

Comment:
I assume you wanted to remove this?

Reply:
Yes. This pr is an amendment to #279, and this now happens inside  [`setupSession`](https://github.com/HedvigInsurance/web-onboarding/pull/279/files#diff-d8aadba1570390110355bbcd32043b06R40-R72) - plus the actual market picker which should happen there also :) unfortunately someone has done quite some copy-pasting of this logic and i missed one place.